ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -91,23 +91,16 @@
             </div>
             <div class="column is-narrow info-table-row-edit">
               {{#if (eq @secretData.length (inc index))}}
-                <button
-                  type="button"
-                  {{action "addRow"}}
-                  class="button is-outlined is-primary"
-                  data-test-secret-add-row="true"
-                >
-                  Add
-                </button>
+                <Hds::Button @text="Add" class="is-outlined" {{action "addRow"}} data-test-secret-add-row="true" />
               {{else}}
-                <button
-                  class="button has-text-grey is-expanded is-icon"
-                  type="button"
+                <Hds::Button
+                  @text="Delete row"
+                  @icon="trash"
+                  @isIconOnly={{true}}
+                  @color="secondary"
+                  class="has-text-grey is-expanded"
```

Comment:
we probably want to remove these classes in case it's creating grey text with a secondary button